app/assets/stylesheets/_work_packages_show_view_overwrite.scss

```diff
@@ -0,0 +1,128 @@
+//-- copyright
+// OpenProject is a project management system.
+// Copyright (C) 2012-2015 the OpenProject Foundation (OPF)
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License version 3.
+//
+// OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+// Copyright (C) 2006-2013 Jean-Philippe Lang
+// Copyright (C) 2010-2013 the ChiliProject Team
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License
+// as published by the Free Software Foundation; either version 2
+// of the License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program; if not, write to the Free Software
+// Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+//
+// See doc/COPYRIGHT.rdoc for more details.
+//++
+
+.work-packages--show-view {
+
+  height: 100%;
+
+  .subject-header {
+    float: left;
+    line-height: 40px;
+    padding: 10px 0 0 0;
+    font-weight: bold;
+    font-size: 1.5rem;
+  }
+
+  .work-packages--split-view {
+    border-top: 1px solid #ccc;
+    height: calc(100% - 62px);
+    overflow: hidden;
+    position: relative;
+
+    .tabrow {
+      margin: -16px -20px 0 -10px;
+    }
+  }
+
+  .work-packages--left-panel {
+    border-right: 1px solid #ccc;
+    overflow-y: auto;
+    overflow-x: auto;
+    width: 60%;
+
+    .work-packages--panel-inner {
+      padding: 0px 20px 20px 0px;
+    }
+  }
+
+  .work-packages--right-panel {
+    .work-packages--panel-inner {
+      padding: 20px 20px 20px 20px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `20px` instead of `20px 20px 20px 20px`